tests/cephfs/snapshot_clone/cg_snap_test.py

```diff
@@ -1580,11 +1580,11 @@ def cg_snap_func_6(cg_test_params):
 # HELPER ROUTINES
 
 
-def wait_for_cg_io(p, qs_id_val, io_run_time):
+def wait_for_cg_io(p, qs_id_val):
     if p.is_alive():
         proc_stop = 0
         log.info("CG IO is running after quiesce lifecycle")
-        wait_time = io_run_time * 20
+        wait_time = 1800
```

Comment:
I think the wait time is too high?

Reply:
Yes neha, i agree, but IO cmds doesn't suceed after quiesce state changes, i had to know whats the long pole for IO cmd to suceed so that i can report on this to Dev